install_instructions/client/raspberry-pi.md

```diff
@@ -1,67 +1,71 @@
+Unlike Server, Stream-Pi Client has two main methods of running on Raspberry Pi and other ARMv7 devices. Desktop Mode and Console Mode.
 
 ## Desktop vs Console Mode
 
-Generally, **_we recommend console mode_** as Desktop mode may result in slower / poorer performance.
+Desktop Mode is to be used when you want to use Stream-Pi as a standard app with its window that you can resize and adjust. 
+
+Console Mode is to be used when you want a clean look without any windowing system.
+
+Generally, **_we recommend console mode_** as Desktop mode may result in slower / poorer performance due to lack of hardware acceleration.
+
+
+**There is a bug in the latest release which prevents some ARMv7 devices like the Raspberry Pi 3 (Raspberry Pi OS) to render icons while running Stream-Pi in console mode. The only solution then is to run Stream-Pi Client in Desktop mode. This issue has been already patched and will no longer be there in future releases.**
 
-## Client Instructions
+
+## Instructions
 
 The below steps have **anchors** associated with them, so you can quickly navigate with the browser's url bar `/install/client/[PLATFORM]#step-1` for example.
 
-### Step 1
+## Step 1
 
 If you downloaded '.zip' file directly from github onto the device you can skip to step 2.
 
-You'll need to get the '.zip' file onto your device, you can always download it directly or you can download on another machine and copy over to this one using like FileZilla, VNC Viewer, or TeamViewer.
+You'll need to get the '.zip' file onto your device, you can always download it directly or you can download on another machine and copy over to this one using like FileZilla, VNC Viewer, Team Viewer, or SCP.
 
-### Step 2
+## Step 2
 
 You'll need to install some additional packages on the system before we continue proper:
 
 - libpangoft2-1.0-0
 - libcairo2-dev
 
-On Raspbian/Debian based distros, open your terminal and run:
+To install them, run:
 
 ```bash
 $ sudo apt install libpangoft2-1.0-0 libcairo2-dev
 ```
 
-### Step 3
+## Step 3
 
 Now that the release '.zip' file is on the device, extract the '.zip' file to a place that is easy for you to remember / easy to access. Also make sure you will always have permission to access this folder.
 
 For example:
 
 ```nginx
-/home/<YOURNAME>/StreamPiClient
+/home/pi/StreamPiClient
 ```
 
-Where `StreamPiClient` holds the unpacked files from the zip. Most programs that unzip will extract its contents to a folder with the same name as the zip file, this is fine too.
-
-What you want to avoid is something like this:
-
-```nginx
-/home/<YOURNAME>/StreamPiClient/StreamPi-FROM-GITHUB
-```
+Be careful to not extract the zip in directories which cant be edited by non-root as it will cause Stream-Pi to not work correctly. For example: `/home/`.
```

Comment:
I don't agree with this change: it seems to be more ambiguous, we should combine this change with what was already there